server/monitor-types/zabbix-trigger.js

```diff
@@ -0,0 +1,64 @@
+const { MonitorType } = require("./monitor-type");
+const { UP, DOWN } = require("../../src/util");
+const axios = require("axios");
+
+class ZabbixTriggerMonitorType extends MonitorType {
+
+    name = "zabbix-trigger";
+
+    /**
+     * @inheritdoc
+     */
+    async check(monitor, heartbeat, _server) {
+
+        // Make Request
+        const options = {
+            method: "post",
+            url: monitor.zabbixInstanceUrl,
+            data: {
+                "jsonrpc": "2.0",
+                "method": "trigger.get",
+                "params": {
+                    "triggerids": monitor.zabbixTriggerId,
+                    "output": "extend",
+                    "selectFunctions": "extend",
+                },
+                // Will be deprecated, but Bearer Auth is currently not working
+                "auth": monitor.zabbixAuthToken,
+                "id": 1
```

Comment:
I am a bit unclear about what the `id` field does. They state that it is

> an arbitrary identifier of the request

=> should we set it to a Random id on each request?